rfcs/0127-issues-warnings.md

```diff
@@ -31,17 +31,19 @@ Apart from that, there is need for a general per-package warning mechanism in ni
 
 A new attribute is added to the `meta` section of a package: `issues`. If present, it is a list of attrsets which each have the following fields:
 
-- `message`: Required. A string message describing the issue with the package. Linking issues and pull requests is encouraged.
+- `message`: Required. A string message describing the issue with the package.
 - `kind`: Optional but recommended. If present, the resulting warning will be printed as `kind: message`.
 - `date`: Required. An ISO 8601 `yyyy-mm-dd`-formatted date from when the issue was added.
+- `urls`: Optional, list of strings. Can be used to link issues, pull requests and other related items.
 
 Example values:
 
 ```nix
 meta.issues = [{
   kind = "deprecated";
-  message = "This package depends on Python 2, which has reached end of life. #148779";
+  message = "This package depends on Python 2, which has reached end of life.";
```

Comment:
I think this "issue" is the prime example where I would expect a transitive warning for an issue we set once on python2.  I'm not sure why you would want to stick it onto a reverse-dependency.

Reply:
The section "Propagation across transitive dependencies" below explains the mechanism behind it in detail. In short, just putting the issue to Python will indeed stop evaluation for all transitive dependencies, but ignoring it will also do so for all of them so users would only get the warning for one affected package.